src/galaxy/app.py

```diff
@@ -1180,12 +1181,12 @@ def run_ogr2ogr_cmd(cmd,binding_file_dir):
             for ele in os.scandir(binding_file_dir):
                 size+=os.path.getsize(ele)
             # print(size/1000000) # in MB
-            if size/1000000 >  12000:
-                logging.warn("Killing ogr2ogr because it exceed 12 GB...")
+            if size/1000000 >  shp_limit:
+                logging.warn(f"Killing ogr2ogr because it exceed {shp_limit} MB...")
                 # process.kill()
                 # os.killpg(os.getpgid(process.pid), signal.SIGTERM)  # Send the signal to all the process groups
-                # shutil.rmtree(binding_file_dir)  
-                raise ValueError("Shapefile Exceed 12 GB Limit")
+                # shutil.rmtree(binding_file_dir)
+                raise HTTPException(status_code=404, detail="Shapefile Exceed {shp_limit} MB Limit")  
```

Comment:
Is it normal there is no formatting character _f_ before the double quote of the detail parameter ?

Reply:
I think it's probably an oversight from @kshitijrajsharma .

Reply:
Thanks for walking through, it's a typo , will fix it !